OWNERS_ALIASES

```diff
@@ -129,6 +129,12 @@ aliases:
     - johnbelamaric
     - klueska
     - pohly
+  wg-etcd-operator-leads:
+    - ahrtr
+    - hakman
+    - jberkus
+    - jmhbnz
+    - justinsb
```

Comment:
@justinsb still waiting for your confirmation :)

Reply:
Justin was in yesterdays sig cl meeting. my suggestion is to add him only  later if he has more time.

Reply:
thx @neolit123

Reply:
removed for now

Reply:
Ah sorry, I'm happy either way!  I do get way too many notifications as it is, but I am planning on participating as a contributor at least!

Reply:
thx for the confirmation. Just added your back @justinsb

Reply:
@ahrtr we also would like to participate in etcd-operator and contribute to this project, even if it will be based on different code base.

@kvaps @sergeyshevch are Kuberentes-SIGs members

@sircthulhu @aobort @lllamnyp @Kirill-Garbar @Uburro are not, should we create PR to join them into SIG's?

Reply:
@kvaps Thanks for your constant interest on this. I think anyone is welcome to participate. We definitely need community help on this project.

Reply:
@kvaps you are free to participate, this however is not a list of members.